### PR TITLE
Remove EL version

### PR DESCRIPTION
**Pull request by maximiliankolb**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.4/Katello 4.6
* [x] Foreman 3.3/Katello 4.5
* [x] Foreman 3.2/Katello 4.4
* [x] Foreman 3.1/Katello 4.3

### CODE REVIEW

File: guides/common/modules/proc_creating-partition-tables.adoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ autopart
 You can also use the *Template* file browser to upload a template file.
 +
 The format of the layout must match that for the intended operating system.
-For example, {RHEL} 7.2 requires a layout that matches a kickstart file.
+For example, {RHEL} requires a layout that matches a kickstart file.
```
Comment (adamlazik1) [inline guides/common/modules/proc_creating-partition-tables.adoc:31]:
```suggestion
For example, {EL} requires a layout that matches a kickstart file.
```
Is RHEL included on purpose or could  we generalize it to EL?

**Review by adamlazik1 — COMMENTED** [review]
Left only one small suggestion, otherwise LGTM

File: guides/common/modules/proc_creating-partition-tables.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_creating-partition-tables.adoc:31]: Applied!

File: guides/common/modules/proc_creating-partition-tables.adoc [hunk 2]
```diff
@@ -15,7 +15,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-partitio
 . Select the *Default* checkbox if you want to set the template to automatically associate with new organizations or locations.
 . Select the *Snippet* checkbox if you want to identify the template as a reusable snippet for other partition tables.
 . From the *Operating System Family* list, select the distribution or family of the partitioning layout.
-For example, {RHEL}, CentOS, and Fedora are in the Red Hat family.
+For example, {EL}, CentOS, and Fedora are in the Red Hat family.
```
Comment (ekohl) [inline guides/common/modules/proc_creating-partition-tables.adoc:18]: I'd say this should stay the way it was. It's listing concrete examples and `Enterprise Linux` doesn't really exist.
```suggestion
For example, {RHEL}, CentOS, and Fedora are in the Red Hat family.
```

Or you can use
```suggestion
For example, {EL} ({RHEL}, CentOS, Rocky, etc) and Fedora are in the Red Hat family.
```

But that's quite long.

Comment (maximiliankolb) [reply guides/common/modules/proc_creating-partition-tables.adoc]: Reverted!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: ✅ 
   1ec365164..9e406e5fe  3.4 -> 3.4
   7adbb0bc1..40588715d  3.3 -> 3.3
   f3bea1e90..3cee1901e  3.2 -> 3.2
   a585a639f..cc91437c5  3.1 -> 3.1

**Pull request merged**